cellprofiler/gui/help.py

```diff
@@ -1493,186 +1493,6 @@
-image. You can display each image using the following modes:
-<ul>
-<li><i>Color:</i> Pixels will have a uniform color which can be selected
-by either clicking on the <i>Color</i> button next to the image name or
-by choosing the image's <i>Color</i> menu entry.</li>
-<li><i>Grayscale:</i> The image will be rendered in shades of gray. The
-color choice will have no effect and the image's <i>Color</i> menu entry will
-be unavailable.</li>
-<li><i>Color map:</i> The image will be rendered using a palette. Your default
-color map will be used initially. To change the color map, select
-the image's <i>Color</i> menu entry from its menu and choose one of the
-color maps from the drop-down. The display will change interactively as you
-change the selection, allowing you to see the image as rendered by your choice.
-Hit <i>OK</i> to accept the new color map or hit <i>Cancel</i> to use the
-color map that was originally selected.</li>
-</ul></p>
-
-<p>The image's <i>Alpha</i> menu entry lets you control the image's transparency.
-This will let you blend colors when the palettes overlap and choose which
-image's intensity has the highest priority. To change the transparency, select
-<i>Alpha</i> from the image's menu. Adjust the transparency interactively
-using the slider bar and hit <i>OK</i> to accept the new value or <i>Cancel</i>
-to restore the value that was originally selected.</p>
-
-<h4>Objects</h4>
-
-<p>You can configure the appearance of objects using the context or <i>Subplots</i>
-menu. Choose the objects you wish to configure from the <i>--- Objects ---</i>
-list in the menu. You will see configuration menu items for the objects'
-display mode, color and alpha value. You can display objects using one of the
-following modes:
-<ul>
-<li><i>Lines:</i> This mode draws a line through the center of each pixel
-that borders the background of the object or another object. It does not
-display holes in the object. The line is drawn using the color shown in
-the <i>Color</i> button next to the objects' name. This option does not
-obscure the border pixels, but can take longer to render, especially if there
-are a large number of objects.</li>
-<li><i>Outlines:</i> This mode displays each pixel in the object's border using
-the color shown in the <i>Color</i> button next to the objects' name. This
-option will display holes in unfilled objects, but the display obscures the
-image underneath the border pixels.</li>
-<li><i>Overlay:</i> This mode displays a different color overlay over each
-object's pixels. Each object is assigned a color using the default color map
-initially. You can choose the color map by
-selecting <i>Color</i> from the objects' menu and choosing one of the available
-color maps. You can change the transparency of the overlay by choosing
-<i>Alpha</i> from the objects' menu.</li>
-</ul></p>
-
-""" % globals()
-
-WV_FIGURE_HELP = tuple(list(FIGURE_HELP) + [(
```

Comment:
Removing `WV_FIGURE_HELP` will break this: https://github.com/CellProfiler/CellProfiler/blob/e9170b747a26a1d546171cbffb6d89604b935e02/cellprofiler/gui/viewworkspace.py#L229

Reply:
Fixed